### DISCUSSION TITLE
Add AI assistant

### DISCUSSION THREAD

**Discussion by makgr** [GrapesJS/grapesjs#5637] [Ideas]
Can you please add an AI assistant to generate a template for a web page or email?

**Comment by illtellyoulater** [comment]
There are already two ongoing discussions on this:

- [ChatGPT assisted edit mode?](https://github.com/GrapesJS/grapesjs/discussions/5558)

- [Enhance Your GrapesJS Projects with ChatGPT Integration](https://github.com/GrapesJS/grapesjs/discussions/5893) which includes an implementation as a plugin
